examples/oneapi/cpp/source/graphs/load_graph.cpp

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright 2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "oneapi/dal/util/load_graph.hpp"
+#include <iostream>
+#include "oneapi/dal/data/graph.hpp"
+#include "oneapi/dal/util/csv_data_source.hpp"
+
+using namespace oneapi::dal;
+using namespace oneapi::dal::preview;
+
+int main(int argc, char **argv) {
+    if (argc < 2)
```

Comment:
Probably we need to add the file in order to really run the example in automation